ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/kubevirt/cucushift-installer-rehearse-aws-ipi-kubevirt-workflow.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/kubevirt/cucushift-installer-rehearse-aws-ipi-kubevirt-workflow.yaml",
```

Comment:
Per https://github.com/openshift/release/pull/35798/files#r1091362272, it is calling `ovn` conf step explicitly, so suggest to move it to `cucushift/installer/rehearse/aws/ipi/ovn` folder.